gpu-deployment-tutorial/gpu-deployment-from-scratch.md

```diff
@@ -0,0 +1,270 @@
+# GPU Deployment Guide 
+Guide Adapted from:[**EuroSciPy 2025**](https://euroscipy.org/schedule/) | [Share Link](https://jacobtomlinson.dev/links/euroscipy-2025)
+
+## Agenda
+- Deployment (40 mins)
+  - NVIDIA Brev
+  - GPU Software Environment Fundamentals
+  - Python packages that use CUDA
+  - Monitoring/debugging tools
+  - Other platforms
+
+## Deployment
+
+The second half of the tutorial will discuss how to get your own GPUs on the cloud in more general terms. In order to dig into some of the things we will be learning we will be launching a VM through the [NVIDIA Brev](https://brev.nvidia.com) portal. 
+
+### Getting set up with Brev
+
+- Sign in to or register an account at https://brev.nvidia.com
+- Ensure you are a member of an organization
+  - One should be created for you when you register, but if not it will say "undefined" in the top right
+  - If you don't have one you can create a new one and give it a name
+- Apply credits to your organization
+  - Navigate to Billing
+  - Select "Redeem Code" <UPDATE IN HERE>
+
+<img width="1398" height="594" alt="Screenshot 2025-08-13 at 15 16 09" src="https://gist.github.com/user-attachments/assets/45f9b71a-eb5f-471f-93fa-2503cf8f4714" />
+
+#### Launching a Brev VM
+
+- Under "GPUs" select "New Instance"
+- Choose a GPU type that costs <$1/hour (e.g an L4) 
```

Comment:
Should we recommend a certain level of ram or cloud? I'm having issues with drivers on AWS, not sure if this is just brev (there is a thread in slack to discuss)